introduction.md

```diff
@@ -0,0 +1,11 @@
+I'm Brad Stokes. I'm a developer and just making sure I have my basics covered.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.